app/shared/src/__tests__/GestureController.test.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+describe('GestureController', () => {
+  it('should call NativeModules.RNKiwiGestureController.enableGestures', () => {
+    const GestureController = require('../GestureController').default;
+
+    GestureController.enableGestures('KiwiHotels');
+    expect(
+      NativeModules.RNKiwiGestureController.enableGestures,
+    ).toHaveBeenCalledWith('KiwiHotels');
+  });
+
+  it('should call noop when there is no native module present', () => {
+    jest.resetModules();
+
+    const OriginalGestureController = NativeModules.RNKiwiGestureController;
+
+    /* $FlowExpectedError: undefined is incompatible with object from NativeModules */
+    NativeModules.RNKiwiGestureController = undefined;
```

Comment:
Obviously it is incompatible with object from native modules, that's why we are suppressing it.

Reply:
I'm deleting now `NativeModules.RNKiwiGestureController`